src/power_grid_model_io/config/excel/vision_en.yaml

```diff
@@ -33,8 +33,12 @@ grid:
       x1: X
       c1: C
       tan1: 0
-      r0: R0
-      x0: X0
+      r0:
+        power_grid_model_io.functions.zeros_to_nan:
+          value: R0
+      x0:
+        power_grid_model_io.functions.zeros_to_nan:
+          value: X0
```

Comment:
This is not correct. We cannot always convert zeros to NaN.

You can have a valid `r0=0` and `x0=1`.

Reply:
Right! Modified the function to align that way.